docs/start/install.mdx

```diff
@@ -97,6 +107,19 @@ Whether you will work locally or in a cloud environment, the first step for all
       pip install 'qiskit[visualization]'
```

Comment:
for the 2 visualisation commands can we combine them into a tab component like we do for the python environment section above? One tab can be "default" and the other can be "zsh"

Reply:
good idea